apps/console/src/features/roles-v1/components/role-list.tsx

```diff
@@ -139,12 +139,20 @@ export const RoleList: React.FunctionComponent<RoleListProps> = (props: RoleList
 
     const { t } = useTranslation();
 
+    const featureConfig: FeatureAccessConfigInterface = useSelector(
```

Comment:
`featureConfig` -> `userRolesV1FeatureConfig`

Reply:
This has already been updated in the latest codebase. https://github.com/wso2/identity-apps/blob/48272abd60e71903e2ff9b52682fdcfa3bdb28e2/features/admin.roles.v1/components/role-list.tsx#L143